src/commands/mod.rs

```diff
@@ -45,25 +45,38 @@ pub struct StartActor {
     pub count: usize,
     /// The name of the model/manifest that generated this command
     pub model_name: String,
+    /// Additional annotations to attach on this command
+    pub annotations: HashMap<String, String>,
 }
 
 from_impl!(StartActor);
 
-impl PartialEq for StartActor {
-    fn eq(&self, other: &StartActor) -> bool {
-        self.reference == other.reference && self.host_id == other.host_id
-    }
-}
-
 impl Hash for StartActor {
     fn hash<H: Hasher>(&self, state: &mut H) {
         self.reference.hash(state);
         self.host_id.hash(state);
+        self.count.hash(state);
+        self.model_name.hash(state);
+
+        for annotation in &self.annotations {
```

Comment:
Going to noodle on this 🤔 We may be able to get away with not hashing annotations if we aren't actually hashing this command anymore (after using Vec<Command>'s)

Reply:
I think, simply, at this point we aren't storing the commands in a HashMap/HashSet anywhere, so we can just remove the derived and implemented traits since we're working with vecs. Will want to revisit if that's the case, but I don't see a reason to change our data structure to a BTreeMap or an IndexMap just to support a use case we don't need right now